plugins/modules/ios_lag_interfaces.py

```diff
@@ -192,14 +198,29 @@
 - name: Override device configuration of all interfaces with provided configuration
   cisco.ios.ios_lag_interfaces:
     config:
-    - name: 20
```

Comment:
Will this break the existing playbooks?

Reply:
Nope, I have added a way to handle the older way of writing the playbook aka Port-Channel20 as 20. This is not right and we should not encourage hence taking it off the examples. But as it was introduced with the module previously it is kept likewise.